doc/website/advanced/iceoryx-on-32-bit.md

```diff
@@ -81,6 +84,9 @@ cmake -S iceoryx_meta -B build-64 -DCMAKE_BUILD_TYPE=Release -DEXAMPLES=ON -DIOX
 cmake --build build-64
 ```
 
+> [!NOTE]
+> On Windows with MSVC, there is now counterpart for `-malign-double` and therefore the 32-64 bit mix-mode does not yet work on Windows.
```

Comment:
there is no counterpart

Reply:
No, not as flag but it seems we are lucky and MSVC puts at least atomics on 8 byte boundaries